### DISCUSSION TITLE
fzf-kill.sh - Interactive interface to search for and kill processes using fzf

### DISCUSSION THREAD

**Discussion by sravioli** [junegunn/fzf#4131] [Show and tell]
Link: <https://gist.github.com/sravioli/9783d22867c36d3f05d1cdee842e8963>

Showcase: 

[![asciicast](https://asciinema.org/a/694701.svg)](https://asciinema.org/a/694701)

**Comment by LangLangBart** [comment]
Will you share the script with the world? I wonder if this header is clickable[^1] to change the sorting.

[^1]: [fzf/CHANGELOG.md at 0.52.0](https://github.com/junegunn/fzf/blob/master/CHANGELOG.md#0520)

**Reply by sravioli** [reply]
ops, forgot to paste the gist link. I've updated the post. The header is sadly not clickable, would be cool though
